pkg/proxy/healthcheck/http.go

```diff
@@ -0,0 +1,42 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package healthcheck
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/golang/glog"
+)
+
+// Satisfies the http.Handler interface needed for each service's http.Server listening port.
+type healthCheckHandler struct {
+	svcName string
```

Comment:
this name is misleading, suggest renaming it to svcNsName or something.
Also suggest a comment, something like: 
```
// A healthCheckHandler serves http requests on /healthz on the service health check node port,
// and responds to every request with the count of endpoints for the given Service on this node.
```